crates/relay/src/benchmarking.rs

```diff
@@ -24,6 +24,9 @@ use vault_registry::{
     Pallet as VaultRegistry,
 };
 
+#[cfg(test)]
+use crate::Pallet as Relay;
```

Comment:
The relay benchmark for `store_block_header` will be quite a t the low end if we assume that no forks/re-orgs are happening. Likely better to include a benchmark that causes a re-org to be on the conservative side. What do you think?

We can also do that later due to timining now.

Reply:
Yep the intent of this PR is to just get the benchmarks to run again, but I used the issue benchmarks to accomplish this.